packages/protvista-variation-adapter/src/tooltipGenerators.ts

```diff
@@ -46,6 +46,20 @@ export const getPopulationFrequencies = (
     )
     .join("")}`;
 
+export const getEnsemblCovidLinks = (variant: Variant) => {
+  const shouldGenerateLink = variant.locations.some(
+    (location) => location.source === "EnsemblViruses"
+  );
+  if (shouldGenerateLink) {
+    const { id } = variant.xrefs.find((xref) => xref.name === "ENA");
+    return (
+      `<h5>Ensembl COVID-19</h5>` +
+      `<p><a href="https://covid-19.ensembl.org/Sars_cov_2/Variation/Explore?v=${id}" target="_blank" rel="noopener noreferrer">${id}</a></p>`
+    );
```

Comment:
I would do
```suggestion
    const xref = variant.xrefs.find((xref) => xref.name === "ENA");
    return xref.id ? (`
      <h5>Ensembl COVID-19</h5>
       <p><a href="https://covid-19.ensembl.org/Sars_cov_2/Variation/Explore?v=${id}" target="_blank" rel="noopener noreferrer">${id}</a></p>
    `) : '';
```
Just in case the xref doesn't exist, or doesn't have an ID

Reply:
Done (but not through suggestion)